src/sparseml/pytorch/torchvision/train.py

```diff
@@ -81,13 +83,24 @@ def train_one_epoch(
         start_time = time.time()
         image, target = image.to(device), target.to(device)
         with torch.cuda.amp.autocast(enabled=scaler is not None):
-            output = model(image)
+            outputs = output = model(image)
             if isinstance(output, tuple):
                 # NOTE: sparseml models return two things (logits & probs)
                 output = output[0]
             loss = criterion(output, target)
 
         if steps_accumulated % args.gradient_accum_steps == 0:
+            if manager is not None:
+                manager.loss_update(
```

Comment:
@rahul-tuli does loss_update return the new loss?

Reply:
<img src="https://media4.giphy.com/media/gmslqIqkiMbgrvBqhZ/giphy.gif"/>